src/index.html

```diff
@@ -7,8 +7,52 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+      type="text/css"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="navigationBar">
+      <a href="#home" class="nav_link">
+        <img src="images/logo.png" alt="icon of the website">
+      </a>
```

Comment:
Here between the `a` and `nav` there should be an empty lines as they are siblings.
Generally there are a few rules regarding spacing your tags for the sake of codes redability:
- siblings should have an empty line between each other
```html
<div>
  <span></span>

  <p></p>
</div>
```
there are however a few exceptions from that rule, for example when you have a bunch of repeated tags like in your ul below:
```html
<div>
  <p></p>
  <p></p>

  <ul>
    <li></li>
    <li></li>
    <li></li>
    <li></li>
  </ul>
</div>
```